### ISSUE TITLE
Rancher 2.2.0-alpha6 cannot edit ingress

### ISSUE THREAD

**Issue by Troyhy** [rancher/rancher#18168]
**bug**

**Steps to reproduce (least amount of steps as possible):**
* create namespace product1
* create workload nginx1 & nginx2
* add ingress to port 80 of nxinx1
* edit ingress. Now namespace is shown incorrectly and you cannot select nginx2

**Result:**
It is not possible to edit ingress, because namespace is wrong

**Other details that may be helpful:**

**Environment information**
- Rancher version v2.2.0-alpha6
- Installation option *single install*

**Cluster information**
- Cluster type Infrastructure Provider: Hetzner with hezner driver
- Machine type VM and specifications (CPU/memory):2vcpu/8g
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.5", GitCommit:"32ac1c9073b132b8ba18aa830f46b77dcceb0723", GitTreeState:"clean", BuildDate:"2018-06-21T11:46:00Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.1", GitCommit:"eec55b9ba98609a46fee712359c7b5b365bdd920", GitTreeState:"clean", BuildDate:"2018-12-13T10:31:33Z", GoVersion:"go1.11.2",Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by loganhz** [comment]
Thanks for the report!
It will be fixed in v2.2.0

**Comment by davidnuzik** [comment]
Version: Master 2.2 (2/21/19)
Confirmed fix.
Follow the steps to reproduce. As you do the namespace will now be listed correctly when you edit the ingress.

![image](https://user-images.githubusercontent.com/45179589/53187717-f90ecb00-35c0-11e9-9c7a-5459e96da8d7.png)
Successfully switched to nginx2 (edit ingress and save changes).

Smoke tested / halo tested this area. Did not identify any issues. HTTP/API appears correct.
